backend/src/routes.ts

```diff
@@ -20,6 +20,10 @@ var config = require('../config/config.js');
 
 var jsonParser = bodyParser.json();
 
+const COOKIE_NAME = 'nappikauppa2';
+const COOKIE_HASH_LENGTH = 8;
```

Comment:
onks tässä ideana, että koko order_hashin tallentaminen cookieen on tietoturvariski, mutta id ei (tietenkään) ole riittävästi tilauksen tietojen hakemiseen? Tuntuu vähän hassulta, mutta ymmärrän että tää on yksinkertainen ratkaisu.

Mitä jos cookieen tallennettaisiin hash of hash, eli sha(order_hash)? Se on selkeästi turvallinen ratkaisu, one-way function puhtaimmillaan.